README.md

```diff
@@ -0,0 +1,45 @@
+# 🌊🛥️ Groovebuoy
+_Minimal server side so that boats can groove_
+
+## What's this?
+
+If you don't know, you better check out what's
+[Grooveboat](https://github.com/stevenleeg/grooveboat).
+
+## Getting started
+
+To run this, you'll need a recent version of
+[node](https://nodejs.org/)
+and
+[yarn](http://gitlab.com/stevenleeg/grooveboat) (or npm) installed.
```

Comment:
this should get swapped out to https://yarnpkg.com/

Reply:
🤯